### ISSUE TITLE
Mac UI Switching & Accessibility

### ISSUE THREAD

**Issue by shatteredsite** [transmission/transmission#5241]
### What is the issue?

The most recent UI upgrade to the MacOS interface has made it far more difficult for those with partial to more severe visual disabilities to use. The color contrast is practically nonexistent and lacks all saturation, the overlapping progress bar is difficult to discern behind the torrent title (a kitschy and poor design in fashion), and the torrents are too close together vertically. The alternating differences in background hue and spacing in the old UI were outstanding in visual differentiation, but have since been discarded. Only text, not visual indicators are now well legible. I only have a partial visual disability that is not too severe (with a large enough monitor), and yet this has left me and several acquaintances without an easy to use, or even usable torrent client on MacOS. Even the built-in accessibility features of MacOS are slow and only compensate so much. Sighted friends are switching as well. Having the ability to switch between the older more legible interface (or the new one modified) and the new one would be a good option for us.

Edit for clarification & grammar.

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.2 (2a57b17031)

**Comment by mikedld** [comment]
See also: #4207.

**Comment by GaryElshaw** [comment]
@shatteredsite Most of the UI changes have been incremental in the last year or so and, probably like most of us contributing, I think the UI has remained pretty consistent. Can you include some screenshots of the previous UI version pinpointing where regressions are?

**Comment by ckerr** [comment]
??
